app/assets/stylesheets/browse.scss

```diff
@@ -334,8 +335,17 @@
   border-radius: 5px;
   padding: 14px 20px;
   margin-bottom: 12px;
+  box-shadow: 2px 30px 60px rgba(0, 0, 0, 0.05);
+}
 
+.align-items-alert {
```

Comment:
you can use directly bootstrap like this `class="d-flex align-items-center"`, this will prevent us from handling a new CSS class.

Reply:
same for align-alert